files/fr/web/html/link_types/preload/index.html

```diff
@@ -0,0 +1,222 @@
+
+
+<h2 id="the_basics">Les bases</h2>
+
+<p>Pour charger un fichier CSS permettant de styler une page, on utilise le plus souvent l'élément <code>&lt;link&gt;</code> de la manière suivante :</p>
+
+<pre class="brush: html">&lt;link rel="stylesheet" href="styles/main.css"&gt;</pre>
+
+<p>Ici, nous allons utiliser la valeur <code>preload</code> sur l'attribut <code>rel</code>, ce qui transformera l'élément <code>&lt;link&gt;</code> en outil de préchargement utilisable pour n'importe quelle ressource. Nous devrons aussi indiquer :</p>
+
+<ul>
+  <li>le chemin de la ressource dans l'attribut {{htmlattrxref("href", "link")}} ;</li>
+  <li>le type de ressource dans l'attribut {{htmlattrxref("as", "link")}} ;</li>
+</ul>
+
+<p>Voici un exemple simple (voir nos <a href="https://github.com/mdn/html-examples/tree/master/link-rel-preload/js-and-css">fichiers JS et CSS d'exemple</a> et le <a href="https://mdn.github.io/html-examples/link-rel-preload/js-and-css/">résultat obtenu</a>) :</p>
+
+<pre class="brush: html">&lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+  &lt;title&gt;Exemple de préchargement JS et CSS&lt;/title&gt;
+
+  &lt;link rel="preload" href="style.css" as="style"&gt;
+  &lt;link rel="preload" href="main.js" as="script"&gt;
+
+  &lt;link rel="stylesheet" href="style.css"&gt;
+&lt;/head&gt;
+
+&lt;body&gt;
+  &lt;h1&gt;Balles rebondissantes&lt;/h1&gt;
+  &lt;canvas&gt;&lt;/canvas&gt;
+
+  &lt;script src="main.js" defer&gt;&lt;/script&gt;
+&lt;/body&gt;</pre>
+
+<p>Dans l'exemple ci-dessus, nous préchargeons nos fichiers CSS et JavaScript afin qu'ils puissent être disponible dès qu'ils sont nécessaire pour le rendu de la page. Cet exemple est trivial, car le navigateur va probablement découvrir en même temps les balises de préchargement, le <code>&lt;link rel="stylesheet"&gt;</code> et le <code>&lt;script&gt;</code>, mais le bénéfice sera bien plus visible si les ressources sont plus nombreuses, plus lourdes et chargées à différents endroits. Par exemple :</p>
+
+<ul>
+  <li>les ressources qui sont chargées depuis un fichier CSS, comme certaines polices et images ;</li>
+  <li>les ressources inclues par des fichiers JavaScript, comme des fichiers JSON, d'autres scripts importés ou des services web ;</li>
+  <li>les fichiers image et vidéos plus importants ;</li>
+</ul>
+
+<p><code>preload</code> dispose aussi d'autres avantages. L'utilisation de l'attribut <code>as</code> pour spécifier le type de contenu à précharger permet au navigateur de :</p>
+
+<ul>
+  <li>prioriser les ressources se chargeant avec davantage de précision ;</li>
+  <li>les stocker dans le cache pour de futures requêtes, ce qui permet de réutiliser les ressources si c'est pertinent ;</li>
+  <li>appliquer la bonne <a href="/fr/docs/Web/HTTP/CSP">stratégie de sécurité du contenu</a> aux ressources ;</li>
+  <li>mettre en place les bons en-têtes de requêtes {{HTTPHeader("Accept")}} pour les ressources ;</li>
```

Comment:
. instead of ; for the last item of the list.

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1808/commits/eac2bca7098ce774dfb3371804dc2bd109b5459b